source/docs/essentials/installation.md

```diff
@@ -0,0 +1,170 @@
+```
+
+It may take a while…
+
+### Configure the project
+
+This skeleton comes with default values for most of the configuration. Below are
+the configurations you will most likely have to change/create.
+
+#### Configure the instance
+
+Front-Commerce uses environment variables for runtime configurations depending
+on the instance that runs (local, staging, production).
+
+Copy the `.env.dist` sample file to a local `.env` and adapt configurations.
+
+#### Configure stores
+
+Configure the project by creating a `src/config/stores.js` file. It should be an
+object with stores codes as keys and configurations as values.
+
+Here is an example:
+
+```js
+module.exports = {
+  default: {
+    locale: "en-GB",
+    currency: "EUR",
+    default_country_id: "GB",
+    localeData: require("react-intl/locale-data/en"),
+    countries: IsoCountries =>
+      IsoCountries.registerLocale(require("i18n-iso-countries/langs/en.json"))
+  }
+};
+```
+
+<blockquote class="info">
+**Magento:** when using Front-Commerce with Magento, store codes
+must match Magento ones.
+</blockquote>
+
+#### Configure remote services
+
+In order for Front-Commerce to interact with remote headless services, you must
+ensure it is configured properly.
+
+For instance, to interact with Magento2 you will need to configure a few things:
+
+- `src/config/serviceKeys/magento2.js`: Magento2 keys (available in the
+  `System > Extensions > Integrations` page of Magento admin area)
+- `src/config/serviceKeys/cache.js`: you will need to ensure that the key
+  configured here is identical to Magento’s `fc_cache_api_token` custom variable
+  to enable cache invalidation from Magento2
+
+We invite you to dig into the other configurations and tweak them if you wish.
+See [our documentation](#TODO) for further information.
+
+#### Optionally configure the caching strategy
+
+To use caching locally, you can configure it in `src/config/caching.js`. For now
+only redis is supported, which requires to have an redis instance to connect to.
+
+See [our caching section](#TODO) to know more about caching in the GraphQL
+middleware.
+
+### Launch the application
+
+Runs your Front-Commerce project in development mode by running:
+
+```sh
+npm run start
+```
+
+Open [http://localhost:4000](http://localhost:4000) to view it in the browser.
+You can also open
+[http://localhost:4000/playground](http://localhost:4000/playground) to explore
+the GraphQL schema and interact with its data.
+
+<blockquote class="info">
+The project will reload automatically in most cases. However, when you are
+trying to override a file that already exists in one of your modules, remember
+to restart this script.
+</blockquote>
+
+### Bonus: launch Storybook, our UI Development Environment
+
+Front-Commerce is component based, and uses
+[Storybook](https://storybook.js.org/) to allow developers to focus on building
+components in isolation.
+
+Storybook is a separate application that will render all the _stories_ written
```

Comment:
It might be a good opportunity to link to the offical `stories` documentation : https://storybook.js.org/basics/writing-stories/